web_monitoring/diffing_server.py

```diff
@@ -67,17 +69,47 @@ def get(self, differ):
         try:
             func = self.differs[differ]
         except KeyError:
-            self.send_error(404)
+            self.send_error(404, reason="Diffing method not found.")
```

Comment:
If we’re improving these messages, one important practice we should follow is to, wherever reasonable, include information about what value was wrong and how it could be fixed.

This message would be even better if were something like:

```py
self.send_error(404, reason=f'Unknown diffing method: `{differ}`. You can get a list of supported differs from the `/` endpoint.')
```